kitsune/search/views.py

```diff
@@ -346,21 +207,17 @@ def simple_search(request, template=None):
         t = 'search/mobile/down.html' if request.MOBILE else 'search/down.html'
         return render(request, t, {'q': cleaned['q']}, status=503)
 
-    items = [(k, v) for k in search_form.fields for
-             v in r.getlist(k) if v and k != 'a']
-    items.append(('a', '2'))
-
     fallback_results = None
     if num_results == 0:
         fallback_results = _fallback_results(language, cleaned['product'])
 
     product = Product.objects.filter(slug__in=cleaned['product'])
     if product:
-        product_titles = [_(p.title, 'DB: products.Product.title')
-                          for p in product]
+        product_titles = [_(p.title, 'DB: products.Product.title') for p in product]
     else:
         product_titles = [_('All Products')]
 
+    # FIXME: This is probably bad l10n.
```

Comment:
We should build something like postatus that shows `# FIXME`, `# TODO` and `# XXX` things. Or maybe that's a terrible idea.